frontend/src/views/camp/CampProgram.vue

```diff
@@ -44,10 +45,15 @@ Show all activity schedule entries of a single period.
             :end="period().end"
             :editable="editMode"
             @newEntry="slotProps.on.newEntry"
+            @unlockReminder="showUnlockReminder"
           />
         </template>
       </template>
     </schedule-entries>
+    <v-snackbar v-model="showReminder" light class="mb-12">
```

Comment:
Why not using a toast? This was the initial idea of toast, so we don't have to pass around events/properties over several levels to show such information to the user.

Reply:
Hm 🤔

The thing is, here it is not just an error, but a hint or a tip. So for me it deserves a different mental model. Because for me the purpose of the toast is to signify "error".

P.S. also if you look closely at the unlock icon in the toolbar it wiggles. How would you solve this, so that we can wiggle the lock 😁